modern/src/map/GeofenceMap.js

```diff
@@ -14,7 +14,7 @@ const GeofenceMap = () => {
     if (response.ok) {
       setGeofences(await response.json());
     }
-  }, []);
+  }, [geofences]); //this will render created or updated geofences when making api calls 
```

Comment:
Why do we need to re-load all geofences?

Reply:
exemple, if we post/put request a circle, it wont render until we reload the page .
this is the only way i figure out how to make it update without reloading

Reply:
So who would update `geofences` then to trigger this?